src/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>React task</title>
+</head>
+<body>
+    <div id="createElement"></div>
+    <div id="component"></div>
+    <div id="pureComponent"></div>
+    <div id="funcComponent"></div>
```

Comment:
When you'll create real app components you should have only 1 root component which will be attached into DOM

Reply:
Yes, I agree. It's just only here.